### DISCUSSION TITLE
Fallback value for column?

### DISCUSSION THREAD

**Discussion by lapidus** [hasura/graphql-engine#8852] [Question] [answered]
I have a table holding translations. All entities have at least `en` (English) column defined but the other can be empty:

```
id, en, sv, es
sweden, Sweden, NULL, Suecia
norway, Norway, NULL, NULL
```

**What are my best options to provide a fallback to English value for a particular language?**

```
{
  table {
      value : sv 
   }
}

=> value: NULL

```

It would be really useful if there was some sort of directive or similar that "coalesces" English onto the value?

```
{
  table {
      value : sv @replaceIfNull(en)
   }
}

=> value: "Sweden"

```

Thanks!

**Comment by tirumaraiselvan** [comment, marked as answer]
You can create a view which selects the COALESCED value:

```
create view translations_with_defaults as
select coalese( sv, en), coalesce(es, en) from translations
```
